vllm/envs.py

```diff
@@ -704,6 +705,10 @@ def maybe_convert_int(value: Optional[str]) -> Optional[int]:
     # It can be changed with this variable if needed for some reason.
     "VLLM_XGRAMMAR_CACHE_MB":
     lambda: int(os.getenv("VLLM_XGRAMMAR_CACHE_MB", "512")),
+
+    # Force disable merge_attn_states CUDA op, use Triton kernel.
+    "VLLM_DISABLE_MERGE_ATTN_CUDA_OP":
+    lambda: bool(int(os.getenv("VLLM_DISABLE_MERGE_ATTN_CUDA_OP", "0"))),
```

Comment:
Is there truly a need for this override? From your benchmarks it seemed like there was no downside in supported cases, so I think we can remove

Reply:
> Is there truly a need for this override? From your benchmarks it seemed like there was no downside in supported cases, so I think we can remove

This environment is for debugging only and can be removed. Done.